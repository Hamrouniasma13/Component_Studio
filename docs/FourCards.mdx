---
name: Four cards (deprecated)
route: /four-cards
menu: 🗑️ Deprecated
---

import { Playground } from 'docz'
import { FourCards, Button } from '../src'

# FourCards

<Playground>
  <FourCards
    title="Mes contrats"
    actions={[
      <Button href="/">First call to action</Button>,
      <Button href="/">Second call to action</Button>,
    ]}
    cards={[
      {
        image: 'https://picsum.photos/546/970?image=670',
        title: 'First title',
        text:
          'Laborum elit nostrud commodo nulla esse non non et officia. Cillum anim veniam pariatur exercitation occaecat in.',
        actions: [
          {
            url: 'https://qwant.com',
            label: 'Call to action',
          },
        ],
      },
      {
        image: 'https://picsum.photos/546/970?image=903',
        title: 'Second title',
        text:
          'Sint voluptate ut irure consectetur sunt quis eiusmod duis cillum duis. Est Lorem cupidatat qui irure sit commodo excepteur adipisicing laboris sit anim cupidatat nulla duis.',
        actions: [
          {
            url: 'https://qwant.com',
            label: 'Call to action',
          },
        ],
      },
      {
        image: 'https://picsum.photos/546/970?image=718',
        title: 'Third title',
        text:
          'Culpa excepteur aute eu occaecat sit aliqua deserunt consectetur.',
        actions: [
          {
            url: 'https://qwant.com',
            label: 'Call to action',
          },
        ],
      },
      {
        image: 'https://picsum.photos/546/970?image=433',
        title: 'Fourth title',
        text:
          'Aliqua et veniam nisi eu incididunt pariatur Lorem consectetur velit fugiat laborum consectetur. Aliqua minim in enim cupidatat sit mollit.',
        actions: [
          {
            url: 'https://qwant.com',
            label: 'Call to action',
          },
        ],
      },
    ]}
  />
</Playground>
