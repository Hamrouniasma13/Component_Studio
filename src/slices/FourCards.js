/** @jsx jsx */
import { jsx, ThemeContext } from '@emotion/core'
import React from 'react'
import PropTypes from 'prop-types'

import Slice from './Slice'
import Text from '../components/Text'
import Button from '../components/Button'

function Card({ image, title, text, actions }) {
  const theme = React.useContext(ThemeContext)

  const styles = {
    wrapper: {
      backgroundColor: 'white',
      borderRadius: 5,
      overflow: 'hidden',
      border: `1px solid ${theme.colors.lightGray}`,
      marginBottom: 30,
      [theme.media.desktop]: {
        flex: '0 1 48%'
      }
    },
    innerWrapper: {
      minHeight: 150,
      height: '100%',
      [theme.media.laptop]: {
        minHeight: 200
      },
      [theme.media.desktop]: {
        display: 'flex'
      }
    },
    image: {
      backgroundImage: `url(${image})`,
      backgroundPosition: 'center',
      backgroundSize: 'cover',
      width: '45%',
      display: 'none',
      [theme.media.desktop]: {
        display: 'block'
      }
    },
    content: {
      padding: 15,
      [theme.media.tablet]: {
        padding: 30
      },
      [theme.media.desktop]: {
        width: '55%'
      }
    },
    action: {
      marginTop: 20
    }
  }

  return (
    <div css={styles.wrapper}>
      <div css={styles.innerWrapper}>
        <div css={styles.image} />

        <div css={styles.content}>
          {title && <Text.h3 mb={15}>{title}</Text.h3>}

          {text && (
            <Text tag="div" mb={30}>
              {text}
            </Text>
          )}

          {actions && actions.length > 0
            ? actions.map(({ url, label }, key) =>
                url && label ? (
                  <div css={styles.action} key={key}>
                    <Button href={url}>{label}</Button>
                  </div>
                ) : null
              )
            : null}
        </div>
      </div>
    </div>
  )
}

Card.propTypes = {
  image: PropTypes.string,
  title: PropTypes.string,
  text: PropTypes.any,
  actions: PropTypes.arrayOf(
    PropTypes.shape({
      url: PropTypes.string,
      label: PropTypes.string
    })
  )
}

export default function FourCards({ title, cards, actions }) {
  const theme = React.useContext(ThemeContext)

  const styles = {
    cards: {
      [theme.media.desktop]: {
        display: 'flex',
        flexWrap: 'wrap',
        justifyContent: 'space-between'
      }
    }
  }

  return (
    <Slice title={title} actions={actions}>
      {cards && cards.length > 0 ? (
        <div css={styles.cards}>
          {cards.map((card, key) => (
            <Card key={key} {...card} />
          ))}
        </div>
      ) : null}
    </Slice>
  )
}

FourCards.propTypes = {
  title: Slice.propTypes.title,
  cards: PropTypes.arrayOf(PropTypes.shape(Card.propTypes)),
  actions: Slice.propTypes.actions
}
