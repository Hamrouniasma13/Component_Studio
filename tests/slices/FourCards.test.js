import React from 'react';
import renderer from 'react-test-renderer';

import FourCards from '../../slices/FourCards';

describe('Component: <FourCards />', () => {
  it('render snapshot', () => {
    const tree = renderer
      .create(
        <FourCards
          title="Mes contrats"
          cards={[
            {
              image: {
                url: 'https://picsum.photos/546/970?image=670',
                alt: 'Achitechture',
                dimensions: { width: 238, height: 380 },
                mobile: {
                  alt: 'architecture',
                  url: 'https://picsum.photos/546/307?image=670',
                  dimensions: { width: 546, height: 307 }
                },
                tablet: {
                  alt: 'architecture',
                  url: 'https://picsum.photos/737/415?image=670',
                  dimensions: { width: 737, height: 415 }
                },
                desktop: {
                  alt: 'architecture',
                  url: 'https://picsum.photos/238/380?image=670',
                  dimensions: { width: 238, height: 380 }
                }
              },
              title: 'Médecine et soins courants',
              text:
                'Généralistes, cardiologue, etc. adhérent au CAS, vous êtes intégralement remboursés [0€ de reste à charge] sauf dans quelques cas.',
              links: [
                {
                  url: 'https://fr.wikipedia.org/wiki/Pyramides_de_Gizeh',
                  label: 'Voir les détails de mes garanties'
                },
                {
                  url: 'https://fr.wikipedia.org/wiki/Pyramides_de_Gizeh',
                  label: 'Estimer mon reste à charge'
                }
              ]
            },
            {
              image: {
                url: 'https://picsum.photos/546/970?image=903',
                alt: 'Achitechture',
                dimensions: { width: 238, height: 380 },
                mobile: {
                  alt: 'architecture',
                  url: 'https://picsum.photos/546/307?image=903',
                  dimensions: { width: 546, height: 307 }
                },
                tablet: {
                  alt: 'architecture',
                  url: 'https://picsum.photos/737/415?image=903',
                  dimensions: { width: 737, height: 415 }
                },
                desktop: {
                  alt: 'architecture',
                  url: 'https://picsum.photos/238/380?image=903',
                  dimensions: { width: 238, height: 380 }
                }
              },
              title: 'Dentaire',
              text:
                'Prise en charge en fonction de l’intervention et de la position de la dent.',
              links: [
                {
                  url: 'https://fr.wikipedia.org/wiki/Pyramides_de_Gizeh',
                  label: 'En savoir plus'
                },
                {
                  url: 'https://fr.wikipedia.org/wiki/Pyramides_de_Gizeh',
                  label: 'Estimer mon reste à charge'
                }
              ]
            },
            {
              image: {
                url: 'https://picsum.photos/546/970?image=718',
                alt: 'Achitechture',
                dimensions: { width: 238, height: 380 },
                mobile: {
                  alt: 'architecture',
                  url: 'https://picsum.photos/546/307?image=718',
                  dimensions: { width: 546, height: 307 }
                },
                tablet: {
                  alt: 'architecture',
                  url: 'https://picsum.photos/737/415?image=718',
                  dimensions: { width: 737, height: 415 }
                },
                desktop: {
                  alt: 'architecture',
                  url: 'https://picsum.photos/238/380?image=718',
                  dimensions: { width: 238, height: 380 }
                }
              },
              title: 'Optique',
              text:
                'Prise en charge à partir de #100€ pour les montures et #160€ pour les verres.',
              links: [
                {
                  url: 'https://fr.wikipedia.org/wiki/Pyramides_de_Gizeh',
                  label: 'En savoir plus'
                },
                {
                  url: 'https://fr.wikipedia.org/wiki/Pyramides_de_Gizeh',
                  label: 'Estimer mon reste à charge'
                }
              ]
            },
            {
              image: {
                url: 'https://picsum.photos/546/970?image=433',
                alt: 'Achitechture',
                dimensions: { width: 238, height: 380 },
                mobile: {
                  alt: 'architecture',
                  url: 'https://picsum.photos/546/307?image=433',
                  dimensions: { width: 546, height: 307 }
                },
                tablet: {
                  alt: 'architecture',
                  url: 'https://picsum.photos/737/415?image=433',
                  dimensions: { width: 737, height: 415 }
                },
                desktop: {
                  alt: 'architecture',
                  url: 'https://picsum.photos/238/380?image=433',
                  dimensions: { width: 238, height: 380 }
                }
              },
              title: 'Hôpital',
              text:
                'Préparez au mieux votre intervention ou celle d’un proche avec notre service Hospiway. ',
              links: [
                {
                  url: 'https://fr.wikipedia.org/wiki/Pyramides_de_Gizeh',
                  label: 'En savoir plus'
                },
                {
                  url: 'https://fr.wikipedia.org/wiki/Pyramides_de_Gizeh',
                  label: 'Estimer mon reste à charge'
                }
              ]
            }
          ]}
        />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('render simpliest fourCards', () => {
    const tree = renderer
      .create(
        <FourCards
          cards={[
            {
              image: {
                url: 'https://picsum.photos/546/970?image=670',
                alt: 'Achitechture',
                dimensions: { width: 238, height: 380 }
              },
              title: 'Médecine et soins courants'
            },
            {
              image: {
                url: 'https://picsum.photos/546/970?image=903',
                alt: 'Achitechture',
                dimensions: { width: 238, height: 380 },
                mobile: {
                  alt: 'architecture',
                  url: 'https://picsum.photos/546/307?image=903',
                  dimensions: { width: 546, height: 307 }
                },
                tablet: {
                  alt: 'architecture',
                  url: 'https://picsum.photos/737/415?image=903',
                  dimensions: { width: 737, height: 415 }
                },
                desktop: {
                  alt: 'architecture',
                  url: 'https://picsum.photos/238/380?image=903',
                  dimensions: { width: 238, height: 380 }
                }
              },
              links: [
                {
                  url: 'https://fr.wikipedia.org/wiki/Pyramides_de_Gizeh',
                  label: 'En savoir plus'
                },
                {
                  url: 'https://fr.wikipedia.org/wiki/Pyramides_de_Gizeh',
                  label: 'Estimer mon reste à charge'
                }
              ]
            },
            {
              image: {
                url: 'https://picsum.photos/546/970?image=718',
                alt: 'Achitechture',
                dimensions: { width: 238, height: 380 },
                mobile: {
                  alt: 'architecture',
                  url: 'https://picsum.photos/546/307?image=718',
                  dimensions: { width: 546, height: 307 }
                },
                tablet: {
                  alt: 'architecture',
                  url: 'https://picsum.photos/737/415?image=718',
                  dimensions: { width: 737, height: 415 }
                },
                desktop: {
                  alt: 'architecture',
                  url: 'https://picsum.photos/238/380?image=718',
                  dimensions: { width: 238, height: 380 }
                }
              },
              title: 'Optique',
              text:
                'Prise en charge à partir de #100€ pour les montures et #160€ pour les verres.'
            },
            {
              image: {
                url: 'https://picsum.photos/546/970?image=433',
                alt: 'Achitechture',
                dimensions: { width: 238, height: 380 },
                mobile: {
                  alt: 'architecture',
                  url: 'https://picsum.photos/546/307?image=433',
                  dimensions: { width: 546, height: 307 }
                },
                tablet: {
                  alt: 'architecture',
                  url: 'https://picsum.photos/737/415?image=433',
                  dimensions: { width: 737, height: 415 }
                },
                desktop: {
                  alt: 'architecture',
                  url: 'https://picsum.photos/238/380?image=433',
                  dimensions: { width: 238, height: 380 }
                }
              },
              title: 'Hôpital',
              text:
                'Préparez au mieux votre intervention ou celle d’un proche avec notre service Hospiway. ',
              links: [
                {
                  url: 'https://fr.wikipedia.org/wiki/Pyramides_de_Gizeh',
                  label: 'En savoir plus'
                },
                {
                  url: 'https://fr.wikipedia.org/wiki/Pyramides_de_Gizeh',
                  label: 'Estimer mon reste à charge'
                }
              ]
            }
          ]}
        />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
