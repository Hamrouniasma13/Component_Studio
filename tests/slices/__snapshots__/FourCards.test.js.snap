// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: <FourCards /> render simpliest fourCards 1`] = `
.emotion-33 {
  margin: 0px auto;
  max-width: 100%;
  padding-left: 15px;
  padding-right: 15px;
  padding-top: 50px;
  padding-bottom: 50px;
}

@media only screen and (min-width :768px) {
  .emotion-33 {
    width: 720px;
  }
}

@media only screen and (min-width :992px) {
  .emotion-33 {
    width: 960px;
  }
}

@media only screen and (min-width :1200px) {
  .emotion-33 {
    width: 1140px;
  }
}

.emotion-0 {
  display: block;
  font-family: Source Sans Pro,Arial,Helvetica,sans-serif;
  margin-bottom: 0;
  font-size: 28px;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
  line-height: 1em;
  font-weight: 600;
  margin-bottom: 30px;
}

@media only screen and (min-width :768px) {
  .emotion-0 {
    font-size: 36px;
    -webkit-letter-spacing: 0.8px;
    -moz-letter-spacing: 0.8px;
    -ms-letter-spacing: 0.8px;
    letter-spacing: 0.8px;
  }
}

@media only screen and (min-width :1200px) {
  .emotion-32 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-flex-wrap: wrap;
    -ms-flex-wrap: wrap;
    flex-wrap: wrap;
    -webkit-box-pack: justify;
    -webkit-justify-content: space-between;
    -ms-flex-pack: justify;
    justify-content: space-between;
  }
}

.emotion-6 {
  position: relative;
  background-color: white;
  border-radius: 5px;
  overflow: hidden;
  border: 1px solid #e5e5e5;
  margin-bottom: 30px;
  min-height: 380px;
}

@media only screen and (min-width :1200px) {
  .emotion-6 {
    -webkit-flex: 0 1 48%;
    -ms-flex: 0 1 48%;
    flex: 0 1 48%;
    height: 380px;
  }
}

@media only screen and (min-width :1200px) {
  .emotion-5 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
  }
}

@media only screen and (min-width :1200px) {
  .emotion-2 {
    width: 45%;
  }
}

.emotion-1 {
  margin: 0 auto;
  display: block;
  width: 100%;
  margin-bottom: 30px;
}

@media only screen and (min-width :1200px) {
  .emotion-1 {
    margin-bottom: 0;
  }
}

.emotion-4 {
  margin: 20px;
  color: #333333;
}

@media only screen and (min-width :1200px) {
  .emotion-4 {
    margin: 0px;
    width: 55%;
    padding: 30px;
  }
}

.emotion-3 {
  display: block;
  font-family: Source Sans Pro,Arial,Helvetica,sans-serif;
  margin-bottom: 0;
  font-size: 18px;
  line-height: 1.5em;
  font-weight: 600;
  margin-bottom: 15px;
}

.emotion-25 {
  font-size: 16px;
  line-height: 1.5;
  font-family: Source Sans Pro,Arial,Helvetica,sans-serif;
  color: black;
  margin-bottom: 100px;
}

.emotion-25 p > a {
  -webkit-text-decoration: underline;
  text-decoration: underline;
  color: #494df4;
  font-weight: bold;
}

@media only screen and (min-width :768px) {
  .emotion-25 {
    font-size: 18px;
    line-height: 1.67;
  }
}

.emotion-11 {
  padding-left: 0;
  position: absolute;
  bottom: 10px;
}

.emotion-9 {
  display: block;
  color: #494df4;
  text-transform: uppercase;
  margin-bottom: 20px;
  font-weight: bold;
  font-size: 13px;
}

.emotion-9:hover {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.emotion-18 {
  font-size: 16px;
  line-height: 1.5;
  font-family: Source Sans Pro,Arial,Helvetica,sans-serif;
  color: black;
  margin-bottom: 40px;
}

.emotion-18 p > a {
  -webkit-text-decoration: underline;
  text-decoration: underline;
  color: #494df4;
  font-weight: bold;
}

@media only screen and (min-width :768px) {
  .emotion-18 {
    font-size: 18px;
    line-height: 1.67;
  }
}

<div
  className="emotion-33"
>
  <h1
    className="emotion-0"
    size="typo2"
  />
  <div
    className="emotion-32"
  >
    <div
      className="emotion-6"
    >
      <div
        className="emotion-5"
      >
        <div
          className="emotion-2"
        >
          <picture>
            <img
              alt="Achitechture"
              className="emotion-1"
              src="https://picsum.photos/546/970?image=670"
            />
          </picture>
        </div>
        <div
          className="emotion-4"
        >
          <h1
            className="emotion-3"
            size="typo5"
          >
            Médecine et soins courants
          </h1>
        </div>
      </div>
    </div>
    <div
      className="emotion-6"
    >
      <div
        className="emotion-5"
      >
        <div
          className="emotion-2"
        >
          <picture>
            <source
              media="(max-width: 576px)"
              srcSet="https://picsum.photos/546/307?image=903"
            />
            <source
              media="(max-width: 1199px)"
              srcSet="https://picsum.photos/737/415?image=903"
            />
            <img
              alt="Achitechture"
              className="emotion-1"
              src="https://picsum.photos/238/380?image=903"
            />
          </picture>
        </div>
        <div
          className="emotion-4"
        >
          <ul
            className="emotion-11"
          >
            <a
              className="emotion-9"
              href="https://fr.wikipedia.org/wiki/Pyramides_de_Gizeh"
              onClick={[Function]}
              target="_blank"
            >
              En savoir plus
            </a>
            <a
              className="emotion-9"
              href="https://fr.wikipedia.org/wiki/Pyramides_de_Gizeh"
              onClick={[Function]}
              target="_blank"
            >
              Estimer mon reste à charge
            </a>
          </ul>
        </div>
      </div>
    </div>
    <div
      className="emotion-6"
    >
      <div
        className="emotion-5"
      >
        <div
          className="emotion-2"
        >
          <picture>
            <source
              media="(max-width: 576px)"
              srcSet="https://picsum.photos/546/307?image=718"
            />
            <source
              media="(max-width: 1199px)"
              srcSet="https://picsum.photos/737/415?image=718"
            />
            <img
              alt="Achitechture"
              className="emotion-1"
              src="https://picsum.photos/238/380?image=718"
            />
          </picture>
        </div>
        <div
          className="emotion-4"
        >
          <h1
            className="emotion-3"
            size="typo5"
          >
            Optique
          </h1>
          <div
            className="emotion-18"
          >
            Prise en charge à partir de #100€ pour les montures et #160€ pour les verres.
          </div>
        </div>
      </div>
    </div>
    <div
      className="emotion-6"
    >
      <div
        className="emotion-5"
      >
        <div
          className="emotion-2"
        >
          <picture>
            <source
              media="(max-width: 576px)"
              srcSet="https://picsum.photos/546/307?image=433"
            />
            <source
              media="(max-width: 1199px)"
              srcSet="https://picsum.photos/737/415?image=433"
            />
            <img
              alt="Achitechture"
              className="emotion-1"
              src="https://picsum.photos/238/380?image=433"
            />
          </picture>
        </div>
        <div
          className="emotion-4"
        >
          <h1
            className="emotion-3"
            size="typo5"
          >
            Hôpital
          </h1>
          <div
            className="emotion-25"
          >
            Préparez au mieux votre intervention ou celle d’un proche avec notre service Hospiway. 
          </div>
          <ul
            className="emotion-11"
          >
            <a
              className="emotion-9"
              href="https://fr.wikipedia.org/wiki/Pyramides_de_Gizeh"
              onClick={[Function]}
              target="_blank"
            >
              En savoir plus
            </a>
            <a
              className="emotion-9"
              href="https://fr.wikipedia.org/wiki/Pyramides_de_Gizeh"
              onClick={[Function]}
              target="_blank"
            >
              Estimer mon reste à charge
            </a>
          </ul>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Component: <FourCards /> render snapshot 1`] = `
.emotion-42 {
  margin: 0px auto;
  max-width: 100%;
  padding-left: 15px;
  padding-right: 15px;
  padding-top: 50px;
  padding-bottom: 50px;
}

@media only screen and (min-width :768px) {
  .emotion-42 {
    width: 720px;
  }
}

@media only screen and (min-width :992px) {
  .emotion-42 {
    width: 960px;
  }
}

@media only screen and (min-width :1200px) {
  .emotion-42 {
    width: 1140px;
  }
}

.emotion-0 {
  display: block;
  font-family: Source Sans Pro,Arial,Helvetica,sans-serif;
  margin-bottom: 0;
  font-size: 28px;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
  line-height: 1em;
  font-weight: 600;
  margin-bottom: 30px;
}

@media only screen and (min-width :768px) {
  .emotion-0 {
    font-size: 36px;
    -webkit-letter-spacing: 0.8px;
    -moz-letter-spacing: 0.8px;
    -ms-letter-spacing: 0.8px;
    letter-spacing: 0.8px;
  }
}

@media only screen and (min-width :1200px) {
  .emotion-41 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-flex-wrap: wrap;
    -ms-flex-wrap: wrap;
    flex-wrap: wrap;
    -webkit-box-pack: justify;
    -webkit-justify-content: space-between;
    -ms-flex-pack: justify;
    justify-content: space-between;
  }
}

.emotion-10 {
  position: relative;
  background-color: white;
  border-radius: 5px;
  overflow: hidden;
  border: 1px solid #e5e5e5;
  margin-bottom: 30px;
  min-height: 380px;
}

@media only screen and (min-width :1200px) {
  .emotion-10 {
    -webkit-flex: 0 1 48%;
    -ms-flex: 0 1 48%;
    flex: 0 1 48%;
    height: 380px;
  }
}

@media only screen and (min-width :1200px) {
  .emotion-9 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
  }
}

@media only screen and (min-width :1200px) {
  .emotion-2 {
    width: 45%;
  }
}

.emotion-1 {
  margin: 0 auto;
  display: block;
  width: 100%;
  margin-bottom: 30px;
}

@media only screen and (min-width :1200px) {
  .emotion-1 {
    margin-bottom: 0;
  }
}

.emotion-8 {
  margin: 20px;
  color: #333333;
}

@media only screen and (min-width :1200px) {
  .emotion-8 {
    margin: 0px;
    width: 55%;
    padding: 30px;
  }
}

.emotion-3 {
  display: block;
  font-family: Source Sans Pro,Arial,Helvetica,sans-serif;
  margin-bottom: 0;
  font-size: 18px;
  line-height: 1.5em;
  font-weight: 600;
  margin-bottom: 15px;
}

.emotion-4 {
  font-size: 16px;
  line-height: 1.5;
  font-family: Source Sans Pro,Arial,Helvetica,sans-serif;
  color: black;
  margin-bottom: 100px;
}

.emotion-4 p > a {
  -webkit-text-decoration: underline;
  text-decoration: underline;
  color: #494df4;
  font-weight: bold;
}

@media only screen and (min-width :768px) {
  .emotion-4 {
    font-size: 18px;
    line-height: 1.67;
  }
}

.emotion-7 {
  padding-left: 0;
  position: absolute;
  bottom: 10px;
}

.emotion-5 {
  display: block;
  color: #494df4;
  text-transform: uppercase;
  margin-bottom: 20px;
  font-weight: bold;
  font-size: 13px;
}

.emotion-5:hover {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

<div
  className="emotion-42"
>
  <h1
    className="emotion-0"
    size="typo2"
  >
    Mes contrats
  </h1>
  <div
    className="emotion-41"
  >
    <div
      className="emotion-10"
    >
      <div
        className="emotion-9"
      >
        <div
          className="emotion-2"
        >
          <picture>
            <source
              media="(max-width: 576px)"
              srcSet="https://picsum.photos/546/307?image=670"
            />
            <source
              media="(max-width: 1199px)"
              srcSet="https://picsum.photos/737/415?image=670"
            />
            <img
              alt="Achitechture"
              className="emotion-1"
              src="https://picsum.photos/238/380?image=670"
            />
          </picture>
        </div>
        <div
          className="emotion-8"
        >
          <h1
            className="emotion-3"
            size="typo5"
          >
            Médecine et soins courants
          </h1>
          <div
            className="emotion-4"
          >
            Généralistes, cardiologue, etc. adhérent au CAS, vous êtes intégralement remboursés [0€ de reste à charge] sauf dans quelques cas.
          </div>
          <ul
            className="emotion-7"
          >
            <a
              className="emotion-5"
              href="https://fr.wikipedia.org/wiki/Pyramides_de_Gizeh"
              onClick={[Function]}
              target="_blank"
            >
              Voir les détails de mes garanties
            </a>
            <a
              className="emotion-5"
              href="https://fr.wikipedia.org/wiki/Pyramides_de_Gizeh"
              onClick={[Function]}
              target="_blank"
            >
              Estimer mon reste à charge
            </a>
          </ul>
        </div>
      </div>
    </div>
    <div
      className="emotion-10"
    >
      <div
        className="emotion-9"
      >
        <div
          className="emotion-2"
        >
          <picture>
            <source
              media="(max-width: 576px)"
              srcSet="https://picsum.photos/546/307?image=903"
            />
            <source
              media="(max-width: 1199px)"
              srcSet="https://picsum.photos/737/415?image=903"
            />
            <img
              alt="Achitechture"
              className="emotion-1"
              src="https://picsum.photos/238/380?image=903"
            />
          </picture>
        </div>
        <div
          className="emotion-8"
        >
          <h1
            className="emotion-3"
            size="typo5"
          >
            Dentaire
          </h1>
          <div
            className="emotion-4"
          >
            Prise en charge en fonction de l’intervention et de la position de la dent.
          </div>
          <ul
            className="emotion-7"
          >
            <a
              className="emotion-5"
              href="https://fr.wikipedia.org/wiki/Pyramides_de_Gizeh"
              onClick={[Function]}
              target="_blank"
            >
              En savoir plus
            </a>
            <a
              className="emotion-5"
              href="https://fr.wikipedia.org/wiki/Pyramides_de_Gizeh"
              onClick={[Function]}
              target="_blank"
            >
              Estimer mon reste à charge
            </a>
          </ul>
        </div>
      </div>
    </div>
    <div
      className="emotion-10"
    >
      <div
        className="emotion-9"
      >
        <div
          className="emotion-2"
        >
          <picture>
            <source
              media="(max-width: 576px)"
              srcSet="https://picsum.photos/546/307?image=718"
            />
            <source
              media="(max-width: 1199px)"
              srcSet="https://picsum.photos/737/415?image=718"
            />
            <img
              alt="Achitechture"
              className="emotion-1"
              src="https://picsum.photos/238/380?image=718"
            />
          </picture>
        </div>
        <div
          className="emotion-8"
        >
          <h1
            className="emotion-3"
            size="typo5"
          >
            Optique
          </h1>
          <div
            className="emotion-4"
          >
            Prise en charge à partir de #100€ pour les montures et #160€ pour les verres.
          </div>
          <ul
            className="emotion-7"
          >
            <a
              className="emotion-5"
              href="https://fr.wikipedia.org/wiki/Pyramides_de_Gizeh"
              onClick={[Function]}
              target="_blank"
            >
              En savoir plus
            </a>
            <a
              className="emotion-5"
              href="https://fr.wikipedia.org/wiki/Pyramides_de_Gizeh"
              onClick={[Function]}
              target="_blank"
            >
              Estimer mon reste à charge
            </a>
          </ul>
        </div>
      </div>
    </div>
    <div
      className="emotion-10"
    >
      <div
        className="emotion-9"
      >
        <div
          className="emotion-2"
        >
          <picture>
            <source
              media="(max-width: 576px)"
              srcSet="https://picsum.photos/546/307?image=433"
            />
            <source
              media="(max-width: 1199px)"
              srcSet="https://picsum.photos/737/415?image=433"
            />
            <img
              alt="Achitechture"
              className="emotion-1"
              src="https://picsum.photos/238/380?image=433"
            />
          </picture>
        </div>
        <div
          className="emotion-8"
        >
          <h1
            className="emotion-3"
            size="typo5"
          >
            Hôpital
          </h1>
          <div
            className="emotion-4"
          >
            Préparez au mieux votre intervention ou celle d’un proche avec notre service Hospiway. 
          </div>
          <ul
            className="emotion-7"
          >
            <a
              className="emotion-5"
              href="https://fr.wikipedia.org/wiki/Pyramides_de_Gizeh"
              onClick={[Function]}
              target="_blank"
            >
              En savoir plus
            </a>
            <a
              className="emotion-5"
              href="https://fr.wikipedia.org/wiki/Pyramides_de_Gizeh"
              onClick={[Function]}
              target="_blank"
            >
              Estimer mon reste à charge
            </a>
          </ul>
        </div>
      </div>
    </div>
  </div>
</div>
`;
